cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="Cypress" />
+
+import { Given, Then } from "cypress-cucumber-preprocessor/steps";
+
+Then('the PDF download link should appear', () => {
+    cy.get(`.ap-file-block`).should('be.visible');
+});
+
+And('a PDF icon should be displayed with a text {string}', (linkName) => {
+    cy.get(`.ap-file-block a[target='_blank']`).should('be.visible');
```

Comment:
this visibility step should  be moved to the step above - `the PDF download link should appear` instead of existing one
Also you are not verifying an actual icon - reach out to Nadia, she has the code ready.

Reply:
added and updated the test for icon